src/data/store.test.jsx

```diff
@@ -2,7 +2,7 @@ import store from './store';
 
 describe('Default store values', () => {
   it('sets courseDetails properly when defined in global', () => {
```

Comment:
The description should probably be updated too! :)

Reply:
Good catch! Thanks!